packages/gatsby/index.d.ts

```diff
@@ -696,15 +696,15 @@ export interface GatsbySSR {
    *   return <Layout {...props}>{element}</Layout>
    * }
    */
-  wrapPageElement?(args: WrapPageElementNodeArgs, options: PluginOptions): any
+  wrapPageElement?(args: WrapPageElementNodeArgs, options: PluginOptions): React.ReactNode
   wrapPageElement?(
     args: WrapPageElementNodeArgs,
     options: PluginOptions
-  ): Promise<any>
+  ): Promise<React.ReactNode>
   wrapPageElement?(
     args: WrapPageElementNodeArgs,
     options: PluginOptions,
-    callback: PluginCallback
+    callback: PluginCallback<void>
```

Comment:
```suggestion
    callback: PluginCallback<React.ReactNode>
```

Reply:
Actually above is wrong - `wrapPageElement` (and other gatsby-ssr / gatsby-browser APIs) shouldn't even have `callback` variant as this is not supported (`callback` is only supported in `gatsby-node` APIs)

Reply:
many other SSR APIs have `callback` variants, should I remove them?